src/data/faq_de.json

```
@@ -146,6 +146,23 @@
                             "Unter dem Reiter Zertifikate kann die App nur '<b>EU Digitale</b> COVID-Zertifikate' scannen.",
                             "Siehe auch folgenden Eintrag zu <a href='#vac_cert_invalid' target='_self' rel='noopener'>ungültigen Impfzertifikaten</a>."
                          ]
+                    },
+                    {
+                        "title": "Wie lange ist mein Zertifikat gültig?​",
+                        "anchor": "eu_dcc_validity",
+                        "active": true,
+                        "textblock": [
+                            "Der folgende Text bezieht sich auf in Deutschland ausgestellte digitale COVID Zertifikate der EU, d.h. Impf-, Test-, und Genesenenzertifikate. Die angegeben Zahlen sind der aktuelle Stand vom Juli 2021.",
+                            "Wir unterscheiden grundsätzlich​<ul><li>die Gültigkeit des digitalen Zertifikats (die <b>technische Gültigkeit</b>) und</li><li>die Gültigkeit des Impfschutzes (die <b>fachliche Gültigkeit</b>) im Sinne der 'Dauer des Schutzes'</li></ul>",
+                            "<b>Technische Gültigkeit</b>",
+                            "Das Zertifikat als Dokument hat ein Ausstellungsdatum und ein technisches Ablaufdatum hat, so wie auch Ihr Reisepass oder Personalausweis beispielsweise. Für alle Zertifikate gilt, dass das Ablaufdatum vom Herausgeber des Zertifikats gesetzt wird. Derzeit werden in Deutschland die digitalen Impf-, Test- und Genesenen-Zertifikate der EU in der Regel mit einer technischen Gültigkeit von einem Jahr ab Ausstellungsdatum ausgestellt.​",

```

Double "hat"
Please cross check the info reg. recovery certificates, I think they also only have a technical validity of 180 days. 

```suggestion
                            "Das Zertifikat als Dokument hat ein Ausstellungsdatum und ein technisches Ablaufdatum, so wie auch Ihr Reisepass oder Personalausweis beispielsweise. Für alle Zertifikate gilt, dass das Ablaufdatum vom Herausgeber des Zertifikats gesetzt wird. Derzeit werden in Deutschland die digitalen Impf-, Test- und Genesenen-Zertifikate der EU in der Regel mit einer technischen Gültigkeit von einem Jahr ab Ausstellungsdatum ausgestellt.​",
```